CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.1.0 (2019-08-14)
+
+- Added the ability for the summary action bar component to be declared inside the split view component. [#16](https://github.com/blackbaud/skyux-action-bars/pull/16)
```

Comment:
This is really minor, but I think this would read more naturally if we rearranged it a bit: "Added the ability to declare the summary action bar component inside the split view component."

Reply:
Done